### ISSUE TITLE
Smaller options

### ISSUE THREAD

**Issue by michaellevy** [eczarny/spectacle#822]
Love this app -- thank you!

Just got a 4k display, and I find myself wanting sixths of a screen (half vertical, third horizontal), and horizontal quarters. Would be great if you could provide those as options, let the user define custom options, or provide "shrink horizontal only" and "shrink vertical only" options.

**Comment by Chatatata** [comment]
Exactly, I'm planning to shoot a pull request regarding that feature.

**Comment by pbauer** [comment]
There are already https://github.com/bpourriahi/spectacle/commit/7bc7f049e133cb5e790c1b37645258bfa7b05bd3 and https://github.com/mlntn/spectacle/commit/7046e72016f0bdebf608178cc2e69ca26063fde0 that seem to add smaller options (I have not tested them). You could try them and if you like them poke the autors for a pull-request.

**Comment by davemkirk** [comment]
I'm using @mlntn's implementation and it works great. @mlntn, please consider submitting a merge request for these changes: https://github.com/mlntn/spectacle/commit/7046e72016f0bdebf608178cc2e69ca26063fde0

**Comment by mlntn** [comment]
@davemkirk Thanks! Glad you found it helpful. I can work on sending a pull request. I changed some labels in English because I didn't think they made sense. I'll probably keep that out of the PR for now.
